README.md

```diff
@@ -90,6 +90,47 @@ Looking for inspiration? You can view the following profiles for an example:
 - [Krupali Trivedi || Chai](https://github.com/EddieHubCommunity/LinkFree/blob/main/data/krupalitrivedi.json)
 - [Pradumna Saraf](https://github.com/EddieHubCommunity/LinkFree/blob/main/data/Pradumnasaraf.json)
```

Comment:
Now we have the testimonials, I think we can remove lines `85` to `91`

Reply:
Should I remove and Update it...

Reply:
yes please